incubator-tools/document_level_accuracy/document_level_accuracy.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
Thanks for pointing out the use of v1. We've considered it, but most of our notebooks primarily uses v1beta3, including utilities functions. Switching to v1 for this specific case causes type mismatch issues due to API differences. To avoid refactoring, we are thinking of sticking with v1beta3 for now.  Please let me know if you still feel a switch to v1 is necessary.